### PR TITLE
fix(ReportsDailyVisits): date format and show all dates

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- change date forma to use month and day like in the example
- change configuration to show all dates when select 3 weeks for example
image

### CODE REVIEW

File: src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js [hunk 1]
```diff
@@ -21,9 +19,8 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       x: {
         type: 'timeseries',
         tick: {
-          format: (x) => {
-            return intl.formatDate(x, { timeZone: 'UTC' });
-          },
+          culling: false,
+          format: '%b %d',
```
Comment (andresmoschini) [inline src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: What is `%b`, does it not depend on current language?

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: %b shows the monthly name with 3 chars

Comment (andresmoschini) [reply src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: But the first chars change between language, right?

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: now yes! 
image

File: src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js [hunk 2]
```diff
@@ -21,8 +21,9 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       x: {
         type: 'timeseries',
         tick: {
+          culling: false,
           format: (x) => {
-            return intl.formatDate(x, { timeZone: 'UTC' });
+            return intl.formatDate(x, { timeZone: 'UTC', month: 'short', day: '2-digit' });
```
Comment (andresmoschini) [inline src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: ❤️

File: src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js [hunk 3]
```diff
@@ -21,8 +21,9 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       x: {
         type: 'timeseries',
         tick: {
+          culling: false,
```
Comment (andresmoschini) [inline src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: What is this?

Comment (fcoronelmakingsense) [reply src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: it's for show all dates in x axis
image

Comment (andresmoschini) [reply src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js]: 👍

**Comment by fcoronelmakingsense** [comment]
test this please

**Pull request merged**